docs/DeveloperGuide.md

```diff
@@ -249,47 +249,228 @@ Priorities: High (must have) - `* * *`, Medium (nice to have)  - `* *`,  Low (un
 
 
 Priority | As a ... | I want to ... | So that I can...
--------- | :-------- | :--------- | :-----------
+-------- | :---------- | :--------- | :-----------
 `* * *` | new user | see usage instructions | refer to instructions when I forget how to use the App
-`* * *` | user | add a new person |
-`* * *` | user | delete a person | remove entries that I no longer need
-`* * *` | user | find a person by name | locate details of persons without having to go through the entire list
-`* *` | user | hide [private contact details](#private-contact-detail) by default | minimize chance of someone else seeing them by accident
-`*` | user with many persons in the address book | sort persons by name | locate a person easily
+`* * *` | user | add a new events (with start and end timings) | keep track of it and complete it in the future
+`* * *` | user | add a new task (tasks that have to be done before a specific deadline) | keep track of the deadline
+`* * *` | user | add a floating task (tasks without specific times) | have a task that can roll over to the next day if I did not get to it
+`* * *` | user | delete a task | remove tasks that I no longer need to do
+`* * *` | user | edit a task | change the details of the tasks
+`* * *` | user | view tasks | decide on the follow-up action for each task
+`* * *` | user | clear all the data | remove all my information
+`* *` | user | prioritize my task | do the more important ones first
+`* *` | user | search for a task by keywords | view the details of task and complete it
+`* *` | user | undo a command | unexecute the last action that I just performed
+`* *` | user | redo a command | redo the last action that I just performed
+`* *` | user | add recurring tasks | save time entering the same task over multiple dates
+`* *` | user | choose my data storage location | have the flexibility to use the program on multiple computers as they can read from the same file stored on the cloud e.g. Google Drive
+`* *` | user | add a tag on tasks | categorize my task
+`* *` | user | edit a tag | rename the tag without the need to delete and add it again
+`* *` | user | mark my tasks as done | indicate that the task has been completed
+`* *` | user | mark my tasks as undone | indicate that the task has not been completed
+`* *` | user | view tasks by tags/priority/date | group my tasks based on a field of my choice
```

Comment:
@jaeoheeail @johnervan @weikangchia @origiri Hey guys, I assume ur PR-ed for me to go through your dev guides rite? But after going through it seems like the suggestions that was given wasn't exactly changed.... so.... what did ur PR=ed for? Do point me to the correct place to eval :) thanks ~